From c9cb56eab08b4b02ed04e987d305170865031a39 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 26 Jul 2021 13:24:09 +0100
Subject: [PATCH 0856/2201] ARM: dts: Correct CM4 PHY MDIO address

The firmware patches the PHY MDIO address in the DTB to cope with
variations between board revisions, but the default for the CM4 PHY
is currently 1 when it should be 0.

Reported-by: Stefan Wahren <stefan.wahren@i2se.com>
Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi-cm4.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
index 9f9a25bb52c3..d9271012a572 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
@@ -277,9 +277,9 @@
 };
 
 &genet_mdio {
-	phy1: ethernet-phy@1 {
+	phy1: ethernet-phy@0 {
 		/* No PHY interrupt */
-		reg = <0x1>;
+		reg = <0x0>;
 	};
 };
 
-- 
2.17.1

